cmd/skaffold/app/cmd/cmd.go

```diff
@@ -38,24 +38,28 @@ var (
 	v            string
 	defaultColor int
 	overwrite    bool
-
-	updateMsg = make(chan string)
 )
 
-var rootCmd = &cobra.Command{
-	Use:   "skaffold",
-	Short: "A tool that facilitates continuous development for Kubernetes applications.",
-}
-
 func NewSkaffoldCommand(out, err io.Writer) *cobra.Command {
+	updateMsg := make(chan string)
+
+	rootCmd := &cobra.Command{
+		Use:           "skaffold",
+		Short:         "A tool that facilitates continuous development for Kubernetes applications.",
+		SilenceErrors: true,
+	}
+
 	rootCmd.PersistentPreRunE = func(cmd *cobra.Command, args []string) error {
 		if err := SetUpLogs(err, v); err != nil {
 			return err
 		}
 		rootCmd.SilenceUsage = true
 		logrus.Infof("Skaffold %+v", version.Get())
+
+		// Avoid race condition
```

Comment:
Can you describe what race condition is it avoiding for dummies like me?

Reply:
I'm going to simplify the code so that this is not needed anymore.